specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json

```diff
@@ -117,22 +117,61 @@
             "type": "integer",
             "format": "int32",
-            "default": 50,
-            "maximum": 100,
-            "minimum": 1
+            "maximum": 2147483647,
+            "minimum": 0
           },
           {
             "in": "query",
             "name": "$skip",
-            "description": "Skip the $skip entries in the collection\r\nWhen both $top and $skip are supplied, $skip is applied first",
+            "description": "$skip indicates the number of records to skip from the list of records held by the server based on the sorting method specified.  By default, we sort by descending start time.\r\n            \r\nClients MAY use $top and $skip query parameters to specify a number of results to return and an offset into the collection.\r\nWhen both $top and $skip are given by a client, the server SHOULD first apply $skip and then $top on the collection.\r\n            \r\nNote: If the server can't honor $top and/or $skip, the server MUST return an error to the client informing about it instead of just ignoring the query options.",
             "type": "integer",
             "format": "int32",
             "default": 0,
             "maximum": 2147483647,
             "minimum": 0
+          },
+          {
+            "in": "query",
+            "name": "$maxpagesize",
+            "description": "$maxpagesize is the maximum items returned in the items in a page.  If more items are requested via $top (or $top is not specified and there are more items to be returned), @nextLink will contain the link to the next page.\r\n            \r\nClients MAY request server-driven paging with a specific page size by specifying a $maxpagesize preference. The server SHOULD honor this preference if the specified page size is smaller than the server's default page size.",
+            "type": "integer",
+            "format": "int32",
+            "default": 50,
+            "maximum": 100,
+            "minimum": 1
+          },
+          {
+            "in": "query",
+            "name": "ids",
+            "description": "Ids to use in filtering",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "status",
+            "description": "Status to use in filtering",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "createdDateTimeUtcStart",
+            "description": "the start datetime to get items after",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "createdDateTimeUtcEnd",
+            "description": "the end datetime to get items before",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "$orderBy",
+            "description": "the sorting query for the collection (ex: 'CreatedDateTimeUtc asc', 'CreatedDateTimeUtc desc')",
```

Comment:
what are the options for this? the properties found on BatchStatusDetail?

Reply:
only CreatedDateTimeUtc asc / desc is supported right now.

Reply:
what's the best way to specify that?

Reply:
Trying to look up prior art and found [this](https://github.com/Azure/azure-rest-api-specs/blob/ba2d430b967bc6299fbedb8dc16ff039e08e1388/specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/artifacts.json#L4032). Although this service doesn't filter/order-by by query param, looks like it's all collapsed into the body of the request. Let me ask around.

Reply:
I know there is only one supported right now, but in the future if more options are added, could a user specify more than one orderBy clause? Wondering if this should be "type": "array" as well.

Reply:
ok, make this an array.